src/FixerFactory.php

```diff
@@ -192,4 +208,51 @@ private function sortFixers()
 
         return $this;
     }
+
+    /**
+     * @param FixerInterface $fixer               fixer to test
+     * @param string[]       $candidateFixerNames fixer names to test against
+     *
+     * @return string[]|null
+     */
+    private function getConflictingFixers(FixerInterface $fixer, array $candidateFixerNames)
+    {
+        $conflictMap = $this->getFixersConflicts($fixer); // to be replaced in the future by calling the Fixer itself for its conflicts
+        if (null === $conflictMap) {
+            return;
+        }
+
+        $conflictingFixers = array();
+        foreach ($candidateFixerNames as $candidate) {
+            if (in_array($candidate, $conflictMap, true)) {
+                $conflictingFixers[] = $candidate;
+            }
+        }
+
+        return count($conflictingFixers) > 0 ? $conflictingFixers : null;
+    }
+
+    /**
+     * @param FixerInterface $fixer
+     *
+     * @return string[]|null
+     */
+    private function getFixersConflicts(FixerInterface $fixer)
+    {
+        static $conflictMap = array(
+                'short_array_syntax' => array('long_array_syntax'),
+                'align_double_arrow' => array('unalign_double_arrow'),
+                'align_equals' => array('unalign_equals'),
+                'concat_with_spaces' => array('concat_without_spaces'),
+                'echo_to_print' => array('print_to_echo'),
+                'long_array_syntax' => array('short_array_syntax'),
+                'no_blank_lines_before_namespace' => array('single_blank_line_before_namespace'),
+                'phpdoc_type_to_var' => array('phpdoc_var_to_type'),
+                'psr0' => array('psr4'),
+        );
+
+        $fixerName = $fixer->getName();
+
+        return array_key_exists($fixerName, $conflictMap) ? $conflictMap[$fixerName] : null;
```

Comment:
I would always return array for simpler implementation and strong typing.
After that `getConflictingFixers` will actually be a one-liner ;)